installer/dev/tracelogging.cpp

```diff
@@ -8,7 +8,20 @@
 
 using namespace WindowsAppRuntimeInstaller::InstallActivity;
 
-// A process-wide callback function for WIL to call each time it logs a failure.
+#define InstallActivity_ExceptionFailFast_StopWithResult() \
```

Comment:
Macro? That tends to be irksome debugging, if/when code changes, yadda yadda

What if this were a function instead?